project/src/main/java/com/google/sps/classes/User.java

```diff
@@ -12,11 +12,11 @@
   private final String school;
   private String phone;
   private String metric;
-  private Boolean admin;
+  private Boolean is_admin;
   private String imageKey;
 
   /*User class is instatiated when a user logins so all their info is accesible without the datatsore being queried each time*/
-  public User(long userId, String name, String email, String password, String birthdate, long studentId, String sex, String school, String phone, String metric, Boolean admin, String imageKey) {
+  public User(long userId, String name, String email, String password, String birthdate, long studentId, String sex, String school, String phone, String metric, Boolean is_admin, String imageKey) {
```

Comment:
Could you keep line shorter? Usually at Google we are using 80 char line

Reply:
done.